woocommerce-gateway-stripe.php

```diff
@@ -81,10 +81,11 @@ function woocommerce_gateway_stripe() {
 	static $plugin;
 
 	if ( ! isset( $plugin ) ) {
-		// Initialize our autoloader.
-		require_once __DIR__ . '/includes/class-wc-stripe-autoloader.php';
-		WC_Stripe_Autoloader::init();
-
+		// Includes the autoloader.
+		$autoload_filepath = __DIR__ . '/vendor/autoload.php';
+		if ( file_exists( $autoload_filepath ) ) {
+			require $autoload_filepath;
```

Comment:
Do we need any fallback here?

Reply:
If we are not removing the class then `WC_Stripe_Autoloader` can be the fallback. 
On second thought, a proper build/zip of the plugin should always contain the `autoload.php` in `vendor` directory. Maybe a fallback is not much needed.

Reply:
I agree. But I added the fallback anyway just to make it safer in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4406/commits/4f1a49738f145179f1284d5d3563865c02bc6349